templates/api/keras_nlp/models/index.md

```diff
@@ -0,0 +1,65 @@
+# KerasNLP Models
+
+KerasNLP contains end-to-end implementations of popular model
+architectures. These models can be created in two ways:
+
+1) Through the `from_preset()` constructor, which instantiates an object with
+a pre-trained configurations, vocabularies, and (optionally) weights. Available
+preset IDs are listed on this page.
+
+2) Through custom configuration controlled by the user. To do this, simply
+pass the desired configuration parameters to the default constructors of the
+symbols documented below.
+
+## Backbone presets
```

Comment:
It may be useful to start with a picture that shows the relationship between classifier, backbone, preprocessor, and tokenizer, since they are the key mental models users need

Reply:
I think for now, I will just link to the getting started guide, which has this.